src/clusterfuzz/_internal/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -2386,6 +2386,50 @@ def test_oom4(self):
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
 
+  def test_centipede_oom(self):
+    """Test centipede's out of memory stacktrace."""
+    os.environ['REPORT_OOMS_AND_HANGS'] = 'True'
+
+    data = self._read_test_data('centipede_oom.txt')
+    expected_type = 'Out-of-memory'
+    expected_address = ''
+    expected_state = 'NULL'
+    expected_stacktrace = data
+    expected_security_flag = False
+
+    self._validate_get_crash_data(data, expected_type, expected_address,
+                                  expected_state, expected_stacktrace,
+                                  expected_security_flag)
+
+  def test_centipede_timeout_enabled(self):
+    """Test a centipede timeout stacktrace (with reporting enabled)."""
+    os.environ['FUZZ_TARGET'] = 'pdfium_fuzzer'
+    os.environ['REPORT_OOMS_AND_HANGS'] = 'True'
```

Comment:
Is report_ooms_and_hangs a centipede specific flag? If not and if it's being tested elsewhere I don't think we need to test again here.

Reply:
Removed the other test (which disables `REPORT_OOMS_AND_HANGS` reporting) and renamed this to avoid confusion : )